src/main/java/com/palantir/refreshable/DefaultRefreshable.java

```diff
@@ -75,6 +78,9 @@ private DefaultRefreshable(T current, Optional<?> strongParentReference, RootSub
         this.current = current;
         this.strongParentReference = strongParentReference;
         this.rootSubscriberTracker = tracker;
+        ReadWriteLock lock = new ReentrantReadWriteLock();
```

Comment:
I didn't put a great deal of thought into this, but I picked unfair because the lock is a replacement for the `synchronized` keyword, which uses unfair locking.

I don't think the overhead of fairness is worthwhile in this case because heavy concurrent usage of a refreshable isn't considered all that heavy for most concurrent datastructures -- the fairness applies to new incoming calls being allowed to race the  AbstractQueueSynchronizer queue processing to overtake the queue, which is a very small interval. I'm not sure if that applies to new readers while a writer is waiting, but that case might not be impacted either way.